webapp/src/Components/Player/Overview/MatchHistory/OverviewMatchHistoryRow.tsx

```diff
@@ -101,11 +114,16 @@ class OverviewMatchHistoryRowComponent extends React.PureComponent<Props> {
                                     {replayScore}
                                 </Typography>
                             </Grid>
-                            <Grid item xs={2}>
+                            <Grid item xs={1}>
```

Comment:
How does this look on mobile

Reply:
https://github.com/SaltieRL/DistributedReplays/pull/320/files/323d678cdc3c5a9a324fc226a1cdbb1344b74b95#diff-a1b0c89b5849680028a2a48469fbd18dR97

Reply:
???

Reply:
ah github is dumb

Reply:
How does it look at mininum sm width then

Reply:
great